apps/acls/serializers/command_acl.py

```diff
@@ -32,7 +32,9 @@ class CommandFilterACLSerializer(BaseSerializer, BulkOrgResourceModelSerializer)
     class Meta(BaseSerializer.Meta):
         model = CommandFilterACL
         fields = BaseSerializer.Meta.fields + ['command_groups']
-        action_choices_exclude = [ActionChoices.notice]
+        action_choices_exclude = [ActionChoices.notice,
+                                  ActionChoices.face_verify,
+                                  ActionChoices.face_online]
 
 
 class CommandReviewSerializer(serializers.Serializer):
```

Comment:
I have reviewed the given Python code and there seem to be no obvious issues or irregularities related to programming concepts, syntax errors, logic flaws (i.e., logical fallacies), misuse of function parameters, etc. However, it would typically benefit from using more specific documentation like comments with descriptions rather than placeholders (`@@`, ````) that could cause confusion when editing.

As for some general suggestions:

* If you need to add further validation, consider writing methods in serializers to validate inputs instead of manually calling Django's default checks.
* Use string formatting in place of python format strings where appropriate as they offer more readability.
* Consider keeping all import directives at top of file unless necessary to define class names directly inside functions/serializers.
  * In addition, ensure each class has a proper docstring explaining its purpose and usage which is important for maintainability even after you decide to change your implementation later on without changing the original classes structure.
- As an example, adding class-level docstrings should follow these conventions: 
```
class MyClass(SomeBaseType): ...
    pass
```

Remember, while reviewing code here I didn't find any major inconsistencies - only minor improvements such as documenting and improving code clarity based on best practices!